source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2594,6 +2594,12 @@ llvm::Optional<uint64_t> SwiftLanguageRuntime::GetByteStride(CompilerType type)
   return {};
 }
 
+llvm::Optional<size_t> SwiftLanguageRuntime::GetBitAlignment(CompilerType type) {
+  if (auto *type_info = GetTypeInfo(type))
+    return type_info->getAlignment();
```

Comment:
Why is this needed and why isn't it calling into SwiftASTContext::GetTypeBitAlign?

Reply:
Sorry, my mistake, this *is* the runtime call in the other function!